src/RequestAPI.d.ts

```diff
@@ -51,6 +51,11 @@ export type RequestAPI = {
 
   "settings::write": <K extends keyof Settings>(key: K, value: any) => void;
   "settings::get": <K extends keyof Settings>(key: K) => Optional<any>;
+  "settings::getos": () => NodeJS.Platform;
+  "settings::close": () => void;
```

Comment:
Since all of these endpoints are Window-related. I want it to be in `window-router.ts` with the `window` prefix. The only exception might be `settings::getOs` other than that `window::close` is a better fit